Data-Structures/treeIntersection/tree-intersection.js

```diff
@@ -0,0 +1,38 @@
+
+const { Node, BinaryTree } = require('../tree/tree')
+
+
+const treeIntersection = (treeOne, treeTwo) => {
+  let allIntersections = []
+
+  const orderTreeOne = treeOne.preOrder(treeOne.root)
+  const orderTreeTwo = treeTwo.preOrder(treeTwo.root)
+
+  let setOne = new Set(orderTreeOne);
+
+  let setTwo = new Set(orderTreeTwo);
```

Comment:
Great job, Susana! 
You used Set as the data structure. I think this is the most appropriate data structure for this code challenge.
Think about making your code even more efficient: how about only using one Set?